mmv1/products/alloydb/api.yaml

```diff
@@ -0,0 +1,286 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Product
+name: Alloydb
+display_name: AlloyDB
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://alloydb.googleapis.com/v1beta/
+scopes:
+  - https://www.googleapis.com/auth/cloud-identity
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: AlloyDB API
+    url: https://console.cloud.google.com/apis/library/alloydb.googleapis.com
+objects:
+  - !ruby/object:Api::Resource
+    name: "Cluster"
+    self_link: "projects/{{project}}/locations/{{location}}/clusters/{{cluster_id}}"
+    base_url: "projects/{{project}}/locations/{{location}}/clusters"
+    create_url: "projects/{{project}}/locations/{{location}}/clusters?clusterId={{cluster_id}}"
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: "A managed alloydb cluster."
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        "AlloyDB": "https://cloud.google.com/alloydb/docs/"
+      api: "https://cloud.google.com/alloydb/docs/reference/rest/v1beta/projects.locations.clusters/create"
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: "name"
+        base_url: "{{op_id}}"
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
+          insert_minutes: 10
+          update_minutes: 10
+          delete_minutes: 10
+      result: !ruby/object:Api::OpAsync::Result
+        path: "response"
+      status: !ruby/object:Api::OpAsync::Status
+        path: "done"
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: "error"
+        message: "message"
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: "clusterId"
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          The ID of the alloydb cluster.
+      - !ruby/object:Api::Type::String
+        name: "location"
+        input: true
+        url_param_only: true
+        description: |
+          The location where the alloydb cluster should reside.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: "name"
+        output: true
+        description: |
+          The name of the cluster resource.
+      - !ruby/object:Api::Type::Time
+        name: "createTime"
+        description: |
+          Time the Cluster was created in UTC.
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: "updateTime"
+        description: |
+          Time the Cluster was updated in UTC.
+        output: true
```

Comment:
```suggestion
```

Let's remove these. Not useful for the end user.

Reply:
I'm fine with leaving these in. it doesn't hurt anything for them to be there, and there may be use cases we aren't foreseeing.

Reply:
Actually, along the lines of the other comments - updateTime can change out of band easily and may not be as useful for that reason.